src/main/java/trackr/model/menu/ItemProfit.java

```diff
@@ -0,0 +1,54 @@
+package trackr.model.menu;
+
+import static java.util.Objects.requireNonNull;
+import static trackr.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents an Item's profit in the Item list.
+ * Guarantees: immutable; is valid as declared in {@link #isValidProfit(String)}
+ */
+public class ItemProfit {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Profit should only contain numbers, and it should be at most 4 digits long";
+    public static final String VALIDATION_REGEX = "^([1-9]{0,4})$";
+    public final String value;
+
+    /**
+     * Constructs an ItemProfit Object
+     */
+    public ItemProfit(ItemPrice itemPrice, ItemCost itemCost) {
+        requireNonNull(itemPrice);
+        requireNonNull(itemCost);
+        this.value = Float.toString(itemPrice.getValue() - itemCost.getValue());
```

Comment:
` this.value = df.format(itemPrice.getValue() - itemCost.getValue());`

Reply:
Thank you I will implement after this PR merges